azure-template/tests/test_simple.py

```diff
@@ -0,0 +1,11 @@
+# the inclusion of the tests module is not meant to offer best practices for
+# testing in general, but rather to support the `find_packages` example in
+# setup.py that excludes installing the "tests" package
+from devtools_testutils import AzureMgmtTestCase
+
+class TemplateTest(AzureMgmtTestCase):
+    def setUp(self):
```

Comment:
Don't see why?

Reply:
I was trying to highlight that these things exist, and should definitely be leveraged. Should I go simpler?